src/NHSD.BuyingCatalogue.Ordering.Api/Settings/PurchaseOrderItem.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Settings
+{
+    public sealed class PurchaseOrderItem
```

Comment:
Same applies to this class as the `PurchaseDocumentSettingsMap`, it can be made internal & moved to the same folder as the associated service / added as a private class.